### ISSUE TITLE
SmartCity-Accessibility #194 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#128]
Build 'SmartCity-Accessibility' is failing!

Last 50 lines of build output:

```
[...truncated 51561 lines...]
lat is : 31.907236 lng is : 34.997891
the name of the restaurant is : Ginot Zvulun St 24, Modi'in-Maccabim-Re'ut, Israel
lat is : 31.909119 lng is : 35.002463
the name of the restaurant is : Hativat Golani St 2-4, Modi'in-Maccabim-Re'ut, Israel
lat is : 31.9072 lng is : 34.997944
the name of the restaurant is : Ginot Zvulun St 24, Modi'in-Maccabim-Re'ut, Israel
lat is : 31.907544 lng is : 34.995997
the name of the restaurant is : Yahalom St 33, Modi'in-Maccabim-Re'ut, Israel
lat is : 31.909119 lng is : 35.002463
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 5.351 sec <<< FAILURE! - in smartcity.accessibility.search.NearbyPlacesTest
nearByPlacesTest(smartcity.accessibility.search.NearbyPlacesTest)  Time elapsed: 5.346 sec  <<< ERROR!
com.teamdev.jxmaps.internal.aq: Uncaught java.lang.NullPointerException: null
	at com.teamdev.jxmaps.internal.ao.a(SourceFile:60)
	at com.teamdev.jxmaps.Geocoder.geocode(SourceFile:40)
	at smartcity.accessibility.search.SearchQuery.Search(SearchQuery.java:72)
	at smartcity.accessibility.search.SearchQuery.searchByCoordinates(SearchQuery.java:107)
	at smartcity.accessibility.mapmanagement.Location.getAddress(Location.java:77)
	at smartcity.accessibility.search.NearbyPlacesTest.nearByPlacesTest(NearbyPlacesTest.java:49)

Running smartcity.accessibility.navigation.NavigationTest
88157718
שדרות תל חי
Tests run: 4, Failures: 0, Errors: 0, Skipped: 3, Time elapsed: 2.91 sec - in smartcity.accessibility.navigation.NavigationTest

Results :

Tests in error: 
  NearbyPlacesTest.nearByPlacesTest:49 » aq Uncaught java.lang.NullPointerExcept...

Tests run: 41, Failures: 0, Errors: 1, Skipped: 4

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:07 min
[INFO] Finished at: 2016-12-31T10:49:33+00:00
[INFO] Final Memory: 28M/201M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.17:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/workspace/SmartCity-Accessibility/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure

```

Changes since last successful build:
- [KoralChapnik] 436253958d54937004f498f9875b29f7d5616c7c - added addressto the nearby test #30

[View full output](http://163.172.171.65:8080/job/SmartCity-Accessibility/194/)

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility' is failing!

Last 50 lines of build output:

```
[...truncated 69403 lines...]
lat is : 31.907236 lng is : 34.997891
the name of the restaurant is : Ginot Zvulun St 24, Modi'in-Maccabim-Re'ut, Israel
lat is : 31.909119 lng is : 35.002463
the name of the restaurant is : Hativat Golani St 2-4, Modi'in-Maccabim-Re'ut, Israel
lat is : 31.9072 lng is : 34.997944
the name of the restaurant is : Ginot Zvulun St 24, Modi'in-Maccabim-Re'ut, Israel
lat is : 31.907544 lng is : 34.995997
the name of the restaurant is : Yahalom St 33, Modi'in-Maccabim-Re'ut, Israel
lat is : 31.909119 lng is : 35.002463
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 5.531 sec <<< FAILURE! - in smartcity.accessibility.search.NearbyPlacesTest
nearByPlacesTest(smartcity.accessibility.search.NearbyPlacesTest)  Time elapsed: 5.528 sec  <<< ERROR!
com.teamdev.jxmaps.internal.aq: Uncaught java.lang.NullPointerException: null
	at com.teamdev.jxmaps.internal.ao.a(SourceFile:60)
	at com.teamdev.jxmaps.Geocoder.geocode(SourceFile:40)
	at smartcity.accessibility.search.SearchQuery.Search(SearchQuery.java:72)
	at smartcity.accessibility.search.SearchQuery.searchByCoordinates(SearchQuery.java:107)
	at smartcity.accessibility.mapmanagement.Location.getAddress(Location.java:77)
	at smartcity.accessibility.search.NearbyPlacesTest.nearByPlacesTest(NearbyPlacesTest.java:49)

Running smartcity.accessibility.navigation.NavigationTest
88157718
שדרות תל חי
Tests run: 4, Failures: 0, Errors: 0, Skipped: 3, Time elapsed: 2.221 sec - in smartcity.accessibility.navigation.NavigationTest

Results :

Tests in error: 
  NearbyPlacesTest.nearByPlacesTest:49 » aq Uncaught java.lang.NullPointerExcept...

Tests run: 41, Failures: 0, Errors: 1, Skipped: 4

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 57.122 s
[INFO] Finished at: 2016-12-31T11:53:33+00:00
[INFO] Final Memory: 28M/201M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.17:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/workspace/SmartCity-Accessibility/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure

```

Changes since last successful build:
- [KoralChapnik] 436253958d54937004f498f9875b29f7d5616c7c - added addressto the nearby test #30

- [Alexander Kaplan] f8d3c7685f33393c536c858f96356e208ec229b9 - some progress on locaiton screen #126



[View full output](http://163.172.171.65:8080/job/SmartCity-Accessibility/195/)

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility' is failing!

Last 50 lines of build output:

```
[...truncated 35310 lines...]
lat is : 31.907236 lng is : 34.997891
the name of the restaurant is : Ginot Zvulun St 24, Modi'in-Maccabim-Re'ut, Israel
lat is : 31.909119 lng is : 35.002463
the name of the restaurant is : Hativat Golani St 2-4, Modi'in-Maccabim-Re'ut, Israel
lat is : 31.9072 lng is : 34.997944
the name of the restaurant is : Ginot Zvulun St 24, Modi'in-Maccabim-Re'ut, Israel
lat is : 31.907544 lng is : 34.995997
the name of the restaurant is : Yahalom St 33, Modi'in-Maccabim-Re'ut, Israel
lat is : 31.909119 lng is : 35.002463
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 5.24 sec <<< FAILURE! - in smartcity.accessibility.search.NearbyPlacesTest
nearByPlacesTest(smartcity.accessibility.search.NearbyPlacesTest)  Time elapsed: 5.238 sec  <<< ERROR!
com.teamdev.jxmaps.internal.aq: Uncaught java.lang.NullPointerException: null
	at com.teamdev.jxmaps.internal.ao.a(SourceFile:60)
	at com.teamdev.jxmaps.Geocoder.geocode(SourceFile:40)
	at smartcity.accessibility.search.SearchQuery.Search(SearchQuery.java:72)
	at smartcity.accessibility.search.SearchQuery.searchByCoordinates(SearchQuery.java:107)
	at smartcity.accessibility.mapmanagement.Location.getAddress(Location.java:77)
	at smartcity.accessibility.search.NearbyPlacesTest.nearByPlacesTest(NearbyPlacesTest.java:49)

Running smartcity.accessibility.navigation.NavigationTest
88157718
שדרות תל חי
Tests run: 4, Failures: 0, Errors: 0, Skipped: 3, Time elapsed: 5.202 sec - in smartcity.accessibility.navigation.NavigationTest

Results :

Tests in error: 
  NearbyPlacesTest.nearByPlacesTest:49 » aq Uncaught java.lang.NullPointerExcept...

Tests run: 41, Failures: 0, Errors: 1, Skipped: 4

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 57.589 s
[INFO] Finished at: 2016-12-31T12:11:28+00:00
[INFO] Final Memory: 28M/202M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.17:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/workspace/SmartCity-Accessibility/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure

```

Changes since last successful build:
- [KoralChapnik] 436253958d54937004f498f9875b29f7d5616c7c - added addressto the nearby test #30

- [Alexander Kaplan] f8d3c7685f33393c536c858f96356e208ec229b9 - some progress on locaiton screen #126

- [Alexander Kaplan] eba7a5c33edbe30dbb281f4c2311eda24ae5c681 - add demo class for showing gui components, currently shows location



[View full output](http://163.172.171.65:8080/job/SmartCity-Accessibility/196/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
